srcpkgs/saml2aws/template

```diff
@@ -1,9 +1,9 @@
 # Template file for 'saml2aws'
 pkgname=saml2aws
-version=2.26.2
+version=2.28.2
 revision=1
 build_style=go
-go_import_path=github.com/Versent/saml2aws
+go_import_path=github.com/Versent/saml2aws/v2
```

Comment:
Thanks @ericonr!

I added the following small change and it works for me. Thank you for helping out with this update!

```suggestion
go_import_path=github.com/versent/saml2aws/v2
```

Reply:
Dang it, thanks for catching!